Packs/IBMMaaS360Security/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,5 @@
+#### Modeling Rules
+
+##### New: IBM MaaS360 Security Modeling Rule
+
+Added IBM MaaS360 modeling rule, which maps event types 'Admin changes audit' and 'Admin login report' into XDM fields.
```

Comment:
Added the IBM MaaS360 modeling rule, which maps event types 'Admin changes audit' and 'Admin login report' into XDM fields.